aten/src/ATen/core/type.cpp

```diff
@@ -143,6 +143,11 @@ std::ostream& operator<<(std::ostream & out, const Type & t) {
   return out;
 }
 
+std::ostream& operator<<(std::ostream& os, SymInt s) {
+  os << "SymInt(" << s.data() << ")";
+  return os;
```

Comment:
I actually wouldn't suggest wrapping in SymInt() unless it actually was symbolic